tests/TPP.Persistence.MongoDB.Tests/Repos/BadgeRepoTest.cs

```diff
@@ -18,12 +19,18 @@ public class BadgeRepoTest : MongoTestBase
         public BadgeRepo CreateBadgeRepo() =>
             new BadgeRepo(CreateTemporaryDatabase(), Mock.Of<IMongoBadgeLogRepo>(), Mock.Of<IClock>());
 
+        internal class MockClock : IClock
+        {
+            public Instant FixedCurrentInstant = Instant.FromUnixTimeSeconds(1234567890);
+            public Instant GetCurrentInstant() => FixedCurrentInstant;
+        }
+
```

Comment:
I'm not sure why I used this class somewhere in the first place. I believe it is redundant because Moq mocks should work fine